onert-micro/luci-interpreter/src/core/RuntimeGraph.cpp

```diff
@@ -417,6 +417,7 @@ uint8_t *RuntimeGraph::getConstDataByTensor(const circle::Tensor *raw_tensor)
 #endif // ENABLE_TRAINING
 
   auto const &buffer = wrap(_reader->buffers()[raw_tensor->buffer()]->data());
+  assert(buffer.data() != nullptr);
```

Comment:
a function 'luci_interpreter::RuntimeGraph::getConstDataByTensor' is dereferenced at While.cpp:110 without checking for null